### DISCUSSION TITLE
Periodic thin green bar at top of LED

### DISCUSSION THREAD

**Discussion by pfmurphy13** [riffnshred/nhl-led-scoreboard#327] [General]
So I've used my LED scoreboard since last season, this season I noticed that periodically a thin green line at the top of the led will pop in. Is this an issue with my LED board or is this a known issue others have experienced?

**Comment by falkyre** [comment]
If it's a full green bar at the top, It means there's an update available. You have started the scoreboard with the --updatecheck command line.  The code will check everyday at 3am to see if there's an update.

**Reply by pfmurphy13** [reply]
Thanks, it seems like this sorted it out
